utils/npm-publish/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lerna/npm-publish",
-  "version": "3.13.0",
+  "version": "3.13.1",
```

Comment:
Unclear why this changed, version fields only change during `lerna version`. Anyway, not the end of the world, it'll just get overwritten during the next release.

Reply:
Ah, I wasn't familiar with how Lerna handled version updates, so I figured I'd make the update manually. This contribution was the first time I've encountered a project using Lerna, so it was all new to me. It seems like a great tool. Thanks for the merge!